app/managers.py

```diff
@@ -0,0 +1,45 @@
+import sqlite3
+
+from app.models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("./cinema.sqlite")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"""
+            INSERT INTO {self.table_name} (first_name, last_name)
+            VALUES (?, ?)
+                """,

```

Comment:
would be nice to move it just 4 spaces to the left :)

Reply:
move what?